docs/src/parallelization.md

```diff
@@ -127,6 +127,10 @@ julia> HDF5.API.set_libraries!("/usr/lib/x86_64-linux-gnu/hdf5/openmpi/libhdf5.s
 ```
 After the preferences are set, restart the Julia REPL again.
 
+!!! note 
+    If multiple MPI installations are present on a system (as is typically the case on a cluster), calling 
+    `MPIPreferences.use_system_binary()` may lead to an undesired selection of the MPI implementation.
```

Comment:
Maybe add a note that one can check, which MPI installation was chosen in the LocalPreferences.toml file?